libs/partners/huggingface/langchain_huggingface/embeddings/huggingface_endpoint.py

```diff
@@ -34,6 +33,10 @@ class HuggingFaceEndpointEmbeddings(BaseModel, Embeddings):
     async_client: Any = None  #: :meta private:
     model: Optional[str] = None
     """Model name to use."""
+    provider: Optional[str] = None
+    """Name of the provider to use for inference with the model specified in `repo_id`.
+        e.g. "cerebras". if not specified, defaults to HF Inference API. 
```

Comment:
```suggestion
        e.g. "sambanova". if not specified, defaults to HF Inference API. 
```

let's document `"sambanova"` here instead since they have 1 embeddings model deployed (Cerebras don't)